src/app.js

```diff
@@ -76,7 +93,9 @@ function loadSchema(schema) {
         var formatPhoneNumber = function () {
             element.value = new parsePhoneNumberFromString(
                 element.value,
-                prompt('Enter a fallback country (as an ISO 3166-1 alpha-2 country code).').toUpperCase()
+                prompt(
+                    'Enter a fallback country (as an ISO 3166-1 alpha-2 country code), e.g. "FR", "GB", or "DE".'
```

Comment:
```suggestion
                    'Enter a fallback country (as an ISO 3166-1 alpha-2 country code), e.g. "fr", "gb", or "de".'
```

This is minor, feel free to disagree, but imho it's a lot easier to type them lowercase (which is also why I added the `.toUpperCase()`.

Reply:
Yeah, I noticed. I prefer to type them in lowercase as well, but thought we might want stick to uppercase examples.